src/CLHEPrandomgenerator.cpp

```diff
@@ -37,6 +37,7 @@
 
 #include "CLHEPrandomgenerator.h"
 #include <CLHEP/Random/Randomize.h>
+#include <CLHEP/Random/Random.h>
```

Comment:
This is not enough, there are still issues blocking from compiling. Please, test.
```
CLHEPrandomgenerator.cpp: In member function 'void CLHEPrandomGenerator::SetSeed(unsigned int)':
CLHEPrandomgenerator.cpp:47:39: error: no matching function for call to 'CLHEP::HepRandom::createInstance(unsigned int&)'
```